package.json

```diff
@@ -49,7 +49,9 @@
     "typescript": "^5.0.4"
   },
   "dependencies": {
-    "@types/geojson": "^7946.0.10"
+    "@types/geojson": "^7946.0.10",
+    "detectincognitojs": "^1.3.0",
```

Comment:
This package is just a single file: https://github.com/Joe12387/detectIncognito/blob/main/src/detectIncognito.ts

We could probably pull it into a local `utils/` file.